config/dev.webpack.config.js

```diff
@@ -12,7 +12,7 @@ const { config: webpackConfig, plugins } = config({
   }),
   ...(process.env.PROXY && {
     useProxy: true,
-    env: 'stage-stable',
+    env: 'stage-beta',
```

Comment:
Do we want this to be pushed?

Reply:
Yeah. It needs the latest chrome. And it will be also obsolete once the router 6 PR is in anyways.